main.go

```diff
@@ -245,6 +259,50 @@ const (
 	LocalRunMode    = "local"
 )
 
+func addMultiClusterEngineWatch(ctx context.Context, uncachedClient client.Client) {
+	for {
+		//_, err := multiclusterengine.GetManagedMCE(ctx, uncachedClient)
+		crdKey := client.ObjectKey{Name: multiclusterengine.Namespace().GetObjectMeta().GetName()}
+		err := uncachedClient.Get(ctx, crdKey, &mcev1.MultiClusterEngine{})
```

Comment:
This is doing a GET for an MCE with the name "multicluster-engine". We don't want to assume the name of any MCE, nor do we care if there is an instance of MCE. We just need to confirm the CRD is applied on the cluster. This is an example of GETting the CRD itself https://github.com/stolostron/backplane-operator/blob/main/pkg/foundation/cluster_manager.go#L97

Reply:
hopefully attempted to call the client.get correctly so it finds the instance of the mce crd instead of the instance of mce